cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -203,6 +205,17 @@ struct sorting_physical_element_comparator {
   }
 };
 
+struct dremel_device_view {
```

Comment:
What does this name "dremel" mean? I haven't come across that word except in the context of [power tools](https://en.wikipedia.org/wiki/Dremel).